src/lib/holocene/button.svelte

```diff
@@ -189,3 +189,9 @@
     {/if}
   </button>
 {/if}
+
+<style lang="postcss">
+  .active {
```

Comment:
@rossedfort looks like we lost `active` styling some where along the way. Would love to discuss best options for adding it back for all variants!